config/locales/en.yml

```diff
@@ -1735,3 +1735,6 @@ en:
       aria_labels:
         expires_at: Member expiration
       inherited_from: Inherited from
+    search_component:
```

Comment:
Since we will pass the placeholder into the search component and will differ depending on the page its being used for, this search placeholder can be moved to groups -> members -> index in the locale file

Reply:
3dbb985ad983a1d56580a16133a6aa71cb67ed2c